modules/ROOT/pages/_partials/release_notes/sync-gateway-3-1-11-release-note.adoc

```diff
@@ -0,0 +1,25 @@
+== 3.1.11 -- September 2024
+
+Version 3.1.11 of Sync Gateway delivers the following features and enhancements:
+
+[#maint-3-1-11]
+=== Fixed Issues
+
+* https://issues.couchbase.com/browse/CBG-4221[CBG-4221 - Pending unused sequences shouldn't update high cache sequence]
```

Comment:
Should these now link to the new JIRA?

https://jira.issues.couchbase.com/browse/CBG-4221
https://jira.issues.couchbase.com/browse/CBG-4218

Reply:
Ooh eck you're right!

Reply:
Done!